docs/developing/octez-client/transactions.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Sending transactions
+authors: Tim McMackin
+lastUpdated: 27 October 2023
+---
+
+Before sending transactions with the Octez client, make sure that you are connected to an RPC node for the network that you want to use.
```

Comment:
it's not a permanent connection to the node, just a configuration parameter, so maybe better (just a suggestion):
```suggestion
Before sending transactions with the Octez client, make sure that your client is configured to use an RPC node for the network that you want to use.
```

Reply:
That's good, it makes this more technically accurate -- thanks!